packages/components/tsconfig.json

```diff
@@ -8,9 +8,7 @@
 			"gutenberg-test-env",
 			"dom-scroll-into-view",
 			"jest",
-			"@testing-library/jest-dom",
-			"snapshot-diff",
-			"@wordpress/jest-console"
```

Comment:
`tsc` would not succeed with these references. Doesn't seem like they're needed, as jest-console is an internal package, and `snapshot-diff` bundles its own types.